### ISSUE TITLE
Evaluation loss keep increasing

### ISSUE THREAD

**Issue by ixtiyoruz** [zhenye234/LLaSA_training#15]
I have trained the model on custom data however the evaluation loss keep increasing. What could be the issue ? Loss is around 0.2 for training. Even though i use training data for testing the loss is huge (more than 10).

**Comment by ixtiyoruz** [comment]
Yeah, i think it is due to data size. When the data size is low, this model cannot learn how to fill the next steps for speech data i think.  The data size i was using was around 20k,  ~ 50 hours of data records for training.

**Comment by koking0** [comment]
> I have trained the model on custom data however the evaluation loss keep increasing. What could be the issue ? Loss is around 0.2 for training. Even though i use training data for testing the loss is huge (more than 10).

@ixtiyoruz Hi, did you train from scratch or fine-tune with lora?

**Comment by koking0** [comment]
@zhenye234 How to output eval loss in code?

**Comment by ixtiyoruz** [comment]
> > I have trained the model on custom data however the evaluation loss keep increasing. What could be the issue ? Loss is around 0.2 for training. Even though i use training data for testing the loss is huge (more than 10).
> 
> [@ixtiyoruz](https://github.com/ixtiyoruz) Hi, did you train from scratch or fine-tune with lora?

just loaded weights and then reset the weights, i beleive model itselg has init weights method which resets weights

**Comment by SeungyounShin** [comment]
<img width="889" height="888" alt="Image" src="https://github.com/user-attachments/assets/0143d142-6bb3-4066-b085-a5c43885d1c7" />

@zhenye234
I am trying to train qwen3-0.6B from scratch with 36k hours of Korean audio but CER after 200k steps not good do you have any idea?
